lib/kafka_ex/consumer_group/manager.ex

```diff
@@ -244,9 +259,14 @@ defmodule KafkaEx.ConsumerGroup.Manager do
 
     # crash the worker if we recieve an error, but do it with a meaningful
     # error message
-    if join_response.error_code != :no_error do
-      raise "Error joining consumer group #{group_name}: " <>
-              "#{inspect(join_response.error_code)}"
+    case join_response do
+      %{error_code: :no_error} -> :ok
+      %{error_code: error_code} ->
+        raise "Error joining consumer group #{group_name}: " <>
+                "#{inspect(error_code)}"
+      {:error, reason} ->
```

Comment:
Same here - dialyzer says that join_response can never be `{:error, _}`

Reply:
Also checked on live application - exception can be raised from line 268